utils.py

```diff
@@ -147,8 +152,20 @@ def iterate_and_order_json(json_data):
     return temp_dict
 
 
+def upload_to_s3(file_path, file_name):
+    """Upload a file to the default S3 bucket"""
```

Comment:
En esta documentacion puedes agregar como se forma la URL del doc porfa!

Reply:
i.e. `the url is compose by folder + name + datetime + etc" i. e. "https://s3.com/bucket/name/folder/etc"`

Reply:
listo :)